autogen/main/variables.tf.tmpl

```diff
@@ -548,8 +536,8 @@ variable "database_encryption" {
   }]
 }
 
-variable "identity_namespace" {
-  description = "Workload Identity namespace. (Default value of `enabled` automatically sets project based namespace `[project_id].svc.id.goog`)"
+variable "workload_pool" {
```

Comment:
I don't think we need to change this variable name (can add a note in the description that this is otherwise known as `workload_pool`).